modules/id5IdSystem.js

```diff
@@ -25,6 +25,19 @@ export const id5IdSubmodule = {
   decode(value) {
     return (value && typeof value['ID5ID'] === 'string') ? { 'id5id': value['ID5ID'] } : undefined;
   },
+
+  // todo: who can confirm the format?
+  ortbFormat: function(data, eidsArray) {
+    if (utils.isStr(data)) {
+      eidsArray.push({
+        source: 'id5id',
```

Comment:
in the original ORTB spec for handling eids (that never actually got implemented, hence still using an extension), the `source` was meant to be the domain of the id provider. In our case, this would be `id5-sync.com`. 

We don't have a strong point of view for whether the source should be `id5id` or `id5-sync.com`, but feel like we should follow a standard approach. Anyone here have an opinion?

Reply:
Thank you @smenzer , I think we should stick to the spec and keep the ```source``` as a domain value.

Reply:
@smenzer I changed the ```source``` to ```id5-sync.com```.